lectures/arma.md

```diff
@@ -170,7 +170,7 @@ By the [Cauchy-Schwartz inequality](https://en.wikipedia.org/wiki/Cauchy%E2%80%9
 
 Evidently, $\gamma(k)$ does not depend on $t$.
 
-### {index}`Wold Representation <single: Wold Representation>`
+### {index}`Wold representation <single: Wold Representation>`
```

Comment:
@longye-tian can you double check if this should be `Wald` or `Wold`?

Reply:
It should be Wold